### DISCUSSION TITLE
Custom network for IPTV

### DISCUSSION THREAD

**Discussion by robbaman** [fabianishere/udm-iptv#153] [Q&A] [answered]
Quick question, in the readme file it says 

    We recommend creating a separate LAN network for IPTV traffic if 
    possible in order to reduce interference of other devices on the network.

If I do, does it matter what VLAN ID to use? Do I break anything if I create a network with VLAN 10 for regular data and VLAN 20 for IPTV?

I'm not an expert so these may be silly questions, but how does the KPN IPTV VLAN ID of 4 play into this? I would guess the IPTV also comes in on the WAN port, but the instructions it doesn't mention any settings to perform with regard to the IPTV VLAN on the WAN/Internet port...

Thanks in advance

**Comment by Bolten88** [comment, marked as answer]
> 

That works fine. I have it configured the same way. But don't forget to change the port profile of the port where the KPN TV box is plugged in.

It does not matter wich VLAN id you apply to the network. The VLAN 4 you are talking about is the VLAN KPN uses on thier network, so on the WAN side of the UDM. You don't have to worry about that. The software of fabian handles that.

![udm](https://user-images.githubusercontent.com/71372668/181278006-b06a86a8-0b19-401f-943e-414c36cdc829.png)
![udm2](https://user-images.githubusercontent.com/71372668/181278160-06c2d421-d91a-4116-a221-b09ec0717f93.png)
